clang-tools-extra/docs/clang-tidy/checks/modernize/use-ranges.rst

```diff
@@ -0,0 +1,24 @@
+.. title:: clang-tidy - modernize-use-ranges
+
+modernize-use-ranges
+====================
+
+Detects calls to standard library iterator algorithms that could be replaced
+with a ranges version instead
+
```

Comment:
Same:
- table with list of what is converted to what
- check options

Reply:
Table/list anything, just so user would know what is actually being replaced by check.